### ISSUE TITLE
Does mmcv support multi-task training?

### ISSUE THREAD

**Issue by mangoyuan** [open-mmlab/mmcv#1319]
Does mmcv support multi-task training e.g. both detection and image-level classification?
If yes, is there any convenient/easy way to implement it?
Thx!

**Comment by zhouzaida** [comment]
hi @mangoyuan , it depends on how you design your model, loss, etc.

A dummy example
```python
class DummyModel(nn.Module):
    def __init__(self, ...):
        self.model = ...
        self.loss1 = ...
        self.loss2 = ...
   
    def train_step(self, x):
        input = x['img']
        label1, label2 = x['labels]
        output1, output2 = self.model(input)
        loss = loss1(output1, label1) + loss2(output2, label2)
        return {'loss': loss'}

# other is the same with the single task
```

**Comment by mangoyuan** [comment]
Thank you for the prompt reply.
